index.html

```diff
@@ -187,6 +187,37 @@ <h4>Skills</h4>
 </section>
 
 
+    <form class="last_section" action="https://formspree.io/f/xbjwkevl" method="post">
+        <div class="logo1">
+        </div>
+        <div class="logo_container">
+            <div class="logo2">
+            </div>
+        
+            <div class="logo3">
+            </div>
+            
+    </div>
+    <div>
+        <h1 class="contact_me">Contact me</h1>
+    </div>
+ 
+        <p class="last_para">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <div>
+   <input type="text" placeholder="Enter your name" maxlength="30" required class="text1"> 
+</div>
+   <input type="email" placeholder="Enter your Email" required class="text1">
+   <div>
+   <textarea class="textarea_form" rows="4" maxlength="500" placeholder="Write your message here" cols="50"></textarea>
```

Comment:
- [ ] Please kindly add `required` attribute here as well

Reply:
@ShoiraTa 
**The following  changes were implemented**
- Add styling to the form 
- fix the background image positioning
- fix the required attribute
- fix the length of the contact form
- Add the hover to the button  
<img width="343" alt="layout_mobile2" src="https://user-images.githubusercontent.com/84595837/159734862-0fc27359-fb81-4406-9ac5-f54a3f3d96b2.png">